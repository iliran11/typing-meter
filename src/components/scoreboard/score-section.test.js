import React from 'react';
import ScoreSection from './score-section';
import { shallow } from 'enzyme';

const props = {
  title: 'WPM',
  score: 20,
  disabled:true,
  specialScoreClass: 'special-class'
}
const wrapper = shallow(<ScoreSection {...props} />);

describe('`ScoreSection`',()=>{
  it ('renders properly',()=>{
    expect(wrapper).toMatchSnapshot();
  })
})