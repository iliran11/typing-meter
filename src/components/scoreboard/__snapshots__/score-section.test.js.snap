// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`ScoreSection\` renders properly 1`] = `
<div
  className="score-section transitionable true undefined"
>
  <h3>
    WPM
  </h3>
  <div
    className="row"
  >
    <i
      className="undefined icon"
    />
    <span
      className="metric-score special-class"
    >
      20
    </span>
  </div>
</div>
`;
